packages/scandipwa/src/store/ProductList/ProductList.dispatcher.ts

```diff
@@ -62,8 +62,6 @@ ProductListDispatcherData
         const { args = {}, isNext } = options;
         const { currentPage = 0 } = args;
 
-        dispatch(updateLoadStatus(false));
```

Comment:
what's the reason for removing the load status reset? was this causing the blink? is the loading state correctly updated without this line?

Reply:
yes, it probably firing from another place in code. checked state with redux dev tool and every thing updating correctly after removing this extra action.